includes/multi-currency/MultiCurrency.php

```diff
@@ -455,6 +456,61 @@ public function get_switcher_widget_markup( array $instance = [], array $args =
 		return ob_get_clean();
 	}
 
+	/**
+	 * Returns the store's current available, enabled, and default currencies.
+	 *
+	 * @return array
+	 */
+	public function get_store_currencies(): array {
+		return [
+			'available' => $this->get_available_currencies(),
+			'enabled'   => $this->get_enabled_currencies(),
+			'default'   => $this->get_default_currency(),
+		];
+	}
+
+	/**
+	 * Gets the currency settings for a single currency.
+	 *
+	 * @param   string $currency_code The currency code to get settings for.
+	 *
+	 * @return  array The currency's settings.
+	 */
+	public function get_single_currency_settings( string $currency_code ): array {
+		$currency_code = strtolower( $currency_code );
```

Comment:
Should we return an error when the currency doesn't exist on the available list?

Reply:
Good catch with this, I have made updates to consider that in https://github.com/Automattic/woocommerce-payments/pull/6468/commits/b95cb5dcaede7900d086aac1daeae14440343dc9